.github/workflows/label-prs.yml

```diff
@@ -0,0 +1,95 @@
+name: Label PR
+
+on:
+  pull_request:
+    types: [assigned, opened, edited, synchronize, reopened]
+
+jobs:
+  validate-branch-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate branch name
```

Comment:
I'm curious whether it makes sense to run this check in CI. At this point, the developer has created a branch and pushed it up. So if the name of that branch is not in the expected format and they see this check failing, the developer would have to close the PR, rename the branch locally, then make a new PR against that branch. I feel like that might lead to a painful process for people who don't know about this (new hires and contributors).

Is it absolutely necessary that PRs get assigned the correct labels? I'm wondering if it would be better to remove this job and update the job below this so that it bails early if it can't read the issue number from the name of the branch.